<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GemBrowser</class>
 <widget class="QMainWindow" name="GemBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GemBrowser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="homeBtn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="go-home"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="BackBtn">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="go-previous"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="forwardBtn">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="go-next"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="address">
         <property name="minimumSize">
          <size>
           <width>581</width>
           <height>0</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>gemini://</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="stopBtn">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="process-stop"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="content">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>11</y>
          <width>751</width>
          <height>411</height>
         </rect>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
        <property name="openLinks">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_all"/>
   </widget>
   <widget class="QMenu" name="menuNavigation">
    <property name="title">
     <string>Navigation</string>
    </property>
    <addaction name="actionGo_to"/>
    <addaction name="actionBack"/>
    <addaction name="actionForward"/>
    <addaction name="separator"/>
    <addaction name="actionHome"/>
    <addaction name="separator"/>
    <addaction name="actionStart"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionIndex"/>
    <addaction name="actionContents"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuNavigation"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="text">
    <string>Select all</string>
   </property>
  </action>
  <action name="actionGo_to">
   <property name="text">
    <string>Go to...</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="text">
    <string>Forward</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionIndex">
   <property name="text">
    <string>Index</string>
   </property>
  </action>
  <action name="actionContents">
   <property name="text">
    <string>Contents</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
